packages/round-manager/src/context/application/BulkUpdateGrantApplicationContext.tsx

```diff
@@ -207,8 +207,11 @@ async function _bulkUpdateGrantApplication({
     const rowsToUpdate = Array.from(
       new Set(
         selectedApplications.map((application) => {
+          let applicationIndex = application.applicationIndex;
+          if (application.applicationIndex === undefined)
+            applicationIndex = Number(application.id.split("-")[1]);
```

Comment:
Any reason not to always use `Number(application.id.split("-")[1])` here, instead of checking if we have an applicationIndex?